code/datums/gods/patrons/forgotten.dm

```diff
@@ -12,3 +12,11 @@
 		"THE SUCCESSORS HALT HIS RETURN!",
 		"PSYDON LIVES!",
 	)
+
+/datum/patron/psydon/can_pray(mob/living/carbon/human/follower)
+	//We just kind of assume the follower is a human here
+	if(istype(follower.wear_neck, /obj/item/clothing/neck/roguetown/psycross))
```

Comment:
It should also check elsewhere, like the wrists... that's where Inquisitors start with theirs. They currently have to awkwardly move it to their necks for a brief moment to pray.